### ISSUE TITLE
Unable to run RPC server on multiple ports with the same IP on the same server

### ISSUE THREAD

**Issue by mtbitcoin** [ethcore/parity#1897]
Prior to the latest pull it was possible to run two different instances of parity on the same machine using the same 127.0.0.1 IP but assigned to a different port. However it is not possible anymore now

Instance 1
-----------
"C:\Program Files\Ethcore\Parity\parity.exe" --tracing on --port 30305 --jsonrpc-interface 127.0.0.1 --jsonrpc-port 8546
.. >> load fine

instances 2
-----------
C:\>"C:\Program Files\Ethcore\Parity\parity.exe" --chain=classic  --jsonrpc-inte
rface 162.220.63.208 --port 30306 --jsonrpc-port 8547
2016-08-09 19:20:05 UTCStarting Parity/v1.3.0-unstable-ab8b763-20160809/x86_64-w
indows-msvc/rustc1.10.0
2016-08-09 19:20:05 UTCUsing state DB journalling strategy fast
2016-08-09 19:20:05 UTCConfigured for Ethereum Classic using Ethash engine
2016-08-09 19:20:05 UTCYour chain is an alternative fork. TRANSACTIONS MAY BE RE
PLAYED ON THE MAINNET!
thread '<unnamed>' panicked at 'called `Result::unwrap()` on an `Err` value: Io(
Error { repr: Os { code: 5, message: "Access is denied." } })', ../src/libcore\r
esult.rs:785
note: Run with `RUST_BACKTRACE=1` for a backtrace.
ERROR: Signer: Could not start listening on specified address. Make sure that no
 other instance is running on Signer's port. Details: Error { repr: Os { code: 1
0048, message: "Only one usage of each socket address (protocol/network address/
port) is normally permitted." } }

**Comment by gavofyork** [comment]
you need to alter the signer/dapps ports, too:

`--signer-port=8181 --dapps-port=8081`

or alternatively turn them off:

`--signer-off --dapps-off`

**Comment by mtbitcoin** [comment]
Thanks, The updated parameters should be "  --no-signer --no-dapps" as per the latest build

**Comment by ferhatmelih** [comment]
I found this answer helpful for some other project I am working with that uses RPC. So I wanted to add this comment for other people who come to solve RPC problem. I also want to add that changing the RPC port from 8080 to 4999 solved my problem. 

[This link from Microsoft](https://support.microsoft.com/en-us/help/908472/how-to-configure-rpc-to-use-certain-ports-and-how-to-help-secure-those) tells that 

>  By default, RPC dynamically allocates ports in the range of 1024 to 5000 for endpoints that do not specify a port on which to listen.

so I tried changing my port address to 4999, 999 1025 etc and it finally worked. 

Thanks again for the answer above. it gave me a clue.
